users/templates/users/privacy.html

```diff
@@ -0,0 +1,48 @@
+{% extends "web/pages/base.html" %}
+
+{% block title %}
+  Настройки конфиденциальности - Melomatch
+{% endblock title %}
+{% block body %}
+  {% include "web/includes/tab.html" %}
+  <div class="columns is-centered">
+    <div class="column is-three-fifths">
+      <div class="my-6">
+        <h3 class="subtitle is-3">Настройка аккаунта</h3>
+        <form method="post">
+          {% csrf_token %}
+          {% for field in form %}
+            <div class="field mt-4">
+              {% if field.name == "is_private" %}
+                <p class="has-text-weight-medium">
+                  Вы можете сделать ваш аккаунт приватным, чтобы другие пользователи не могли его просматривать.
+                </p>
+              {% else %}
+                <p class="has-text-weight-medium mt-5">
+                  Вы можете активировать ссылку для сравнения музыки, чтобы легче находить людей с похожим вкусом.
```

Comment:
Тут тоже чуть-чуть другой смысл)
```suggestion
                  Вы можете активировать ссылку для сравнения музыки, чтобы другие пользователи могли сравнить свой музыкальный вкус с вашим, пройдя по ней.
```

Reply:
Получается тавтология. Предлагаю заменить на:
Вы можете активировать ссылку для сравнения треков, чтобы другие пользователи могли сопоставить свой музыкальный вкус с вашим, пройдя по ней.

Reply:
Окей, согласен)